sdks/python/apache_beam/transforms/util.py

```diff
@@ -223,7 +223,7 @@ def __init__(self,
     if target_batch_duration_secs and target_batch_duration_secs <= 0:
       raise ValueError("target_batch_duration_secs (%s) must be positive" % (
           target_batch_duration_secs))
-    if max(0, target_batch_overhead, target_batch_duration_secs) == 0:
+    if not (target_batch_overhead or target_batch_duration_secs):
```

Comment:
What is the reason for this change?

I do not think this is checking for at least one number being > 0. For example not (-1 or -3) will not raise this error.

Reply:
These are changes introduced in #7022. This PR is rebased on top of that one, since those changes have an impact on the testing module.

Reply:
Previous if checks make sure those numbers are non-negative.